src/main/java/mate/academy/Main.java

```diff
@@ -11,27 +11,23 @@ public class Main {
     public static final Injector injector = Injector.getInstance("mate.academy");
 
     public static void main(String[] args) {
-        Book book = new Book();
-        book.setTitle("Test Book");
-        book.setPrice(BigDecimal.valueOf(230));
-        Book book2 = new Book();
-        book2.setTitle("Test Book2");
-        book2.setPrice(BigDecimal.valueOf(530));
 
         BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
-        bookDao.create(book);
-        bookDao.create(book2);
+        createBook(bookDao, "Test Book", BigDecimal.valueOf(230));
+        createBook(bookDao, "Test Book2", BigDecimal.valueOf(530));
+
         List<Book> allBooks = bookDao.findAll();
         System.out.println("first list of books" + allBooks);
 
-        book.setTitle("Test Book Update");
-        book.setId(8L);
+        Book bookToUpdate = new Book();
```

Comment:
After correcting createBook method (so that it will return Book object) you can use it here.